datagenerator/src/main/resources/rythm/record.rythm

```diff
@@ -84,6 +86,16 @@ namespace @cSharpRecord.getNamespace()
 }
     }
 
+    public Dictionary<string, object> Data()
+    {
+      Dictionary<string, object> dataMap = new Dictionary<string, object>();
+@for (Field field : cSharpRecord.getFields()) {
+      if (has@field.getName().capFirst())
```

Comment:
This boolean field name should be encapsulated in CSharpRecord.Field class. This way later on if we rename that field name, we don't need to change all these rythm templates.